chamber/storages/boto3.py

```diff
@@ -46,6 +47,13 @@ def _clean_name(self, name):
         # pathlib support
         return super()._clean_name(str(name))
 
+    def _open(self, name, mode='rb'):
+        try:
+            return super()._open(name, mode='rb')
+        except ClientError as ex:
+            if ex.response['Error']['Code'] == '403':
+                raise PermissionError(f'Cannot open file "{name}": {ex.response["Error"]["Message"]}')
```

Comment:
Where is imported PermissionError I cannot find it.

Reply:
It's a built-in exception. https://docs.python.org/3/library/exceptions.html#PermissionError

Reply:
You are right I miss this exception with python 3 upgrade from python 2. It make sense

You should reraise the client error if exception is not 403.

Reply:
I added exception reraising